TravelerApp/TravelerAppCore/Controller/AddHotelInfo.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TravelerAppCore.Models.Hotels;
+
+namespace TravelerAppCore.Controller
+{
+    public static class AddHotelInfo
```

Comment:
klasa AddHotelInfo zastanów się czy taka klasa jest zgodna z przeznaczeniem klas. Nawet po samej nazwie widać że to raczej powinna być jakaś metoda

Reply:
Faktycznie, zgadzam się. Obecnie do tego celu stworzona jest metoda CreateNew w klase Hotel.